### PR TITLE
update the readme file with the installation step

### PR DESCRIPTION
**Pull request by sakthivel9963**

### CODE REVIEW

**Comment by gokatz** [comment]
I'm away from my desktop. Will review this soon.

**Comment by gokatz** [comment]
Hi, @sakthivel9963. I can see that your changes are more relevant to contribution but with the actual installation of the extension. Please refer these packages for Installation section.

* https://github.com/ovity/octotree#install-octotree
* https://github.com/sindresorhus/refined-github#install

Thanks for your efforts. However, the contribution guides also need to be updated and this #17 tracks the same. If you want, help him with the changes.

**Comment by sakthivel9963** [comment]
I can add the link to the chrome and firefox extension.

**Comment by sakthivel9963** [comment]
Can in get the link for the extensions or it should be installed manually.

**Comment by gokatz** [comment]
Oops. Forgot to link them. Use this for Firefox: https://addons.mozilla.org/en-US/android/addon/user-agent-parser/

And Chrome's link needs to be discussed. use the same link for chrome too. I'll change that later

**Comment by sakthivel9963** [comment]
update the readme file with the chrome and firefox extensions.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,48 @@
 # UA Parser - Browser Extension
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
 
+## Prerequisites
+
+1. **NodeJS:** <br>
+   Please install [NodeJS >= 10.15.0](https://nodejs.org/en/download/). If you already have it, you're good to go.
+   <br>
+1. **Yarn:** <br>
```
Comment (gokatz) [inline README.md]: can we explicitly mention as list item `2` ?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+1. [Clone](https://help.github.com/articles/cloning-a-repository/) this repository from GitHub onto your local computer.
+
+   ```sh
+   $ git clone https://github.com/gokatz/ua-parser-extension.git
+   ```
+
+1. Navigate into the project folder.
+
+   ```sh
+   cd ua-parse-extension
```
Comment (gokatz) [inline README.md]: Small typo:

```suggestion
   cd ua-parser-extension
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,48 @@
 # UA Parser - Browser Extension
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
 
+## Prerequisites
```
Comment (gokatz) [inline README.md]: can we move the `Prerequisites` section inside the contribution as it has nothing to do with the actual installation?

**Review by gokatz — CHANGES_REQUESTED** [review]
Added a few comments. Please have a look and let me know.

**Comment by sakthivel9963** [comment]
I will change the all your request and raise the pull request.

File: README.md (hunk 1 above)
Comment (sakthivel9963) [reply README.md]: change according to the requirement.

**Comment by sakthivel9963** [comment]
I have change the code. Please review it

**Comment by sakthivel9963** [comment]
> Added a few comments. Please have a look and let me know.

I have change the code.

**Pull request merged**